### ISSUE TITLE
Frenzy is always on

### ISSUE THREAD

**Issue by grendel111111** [javierriveracastro/betteroll-swade#339]
Frenzy has the code (defaultChecked:"on") so it is always on if you have the edge. But it only applies to the first attack each turn. Maybe remove it?

**Comment by javierriveracastro** [comment]
I'm not sure... you usually want to use it. Anyway done.

You can just edit "builtin-actions,js" here (in github) and ask for a pull request here if you want to suggest this kind of changes in the future. You are probably more proficient with global actions now than myself ;P.

https://github.com/javierriveracastro/betteroll-swade/blob/version_2/betterrolls-swade2/scripts/builtin-actions.js

**Comment by SalieriC** [comment]
There really is little reason to not use it. I'd leave it to default on. In the rare(er) case of a multi action attack, disabling it is less of a hassle than enabling it each time you're attacking.
Reopening it until we are done discussing the matter.

**Comment by javierriveracastro** [comment]
But, because of human nature, it is always easier for a player to remember activating something beneficial that remember deactivating it.

**Comment by SalieriC** [comment]
That may be so, but at the same time, humans are also adaptive and thus able to adjust their behaviour. I'd say we should default to a positive idea of mankind here.

**Comment by javierriveracastro** [comment]
We are talking about RPG PLAYERS, not mankind. Players are evil. I know it, sometimes I do play ;P.

**Comment by SalieriC** [comment]
I get you're joking. It might as well hold some truth though. But the vast majority will be able to appreciate the lack of extra steps they need to take. And even if not, The GM certainly will. I frown when I think of remembering to activate it every time I make an attack with a certain actor.

**Comment by javierriveracastro** [comment]
Let's wait for grendel1111 input. Note that this conversation is not only about this bug, it will probably set a policy for other things (like hindrances), so jokes or not, the conclusion should be sound.

**Comment by grendel111111** [comment]
I noticed this because it was the only Edge coming up red. When I was making my actions I first decided to go with edges are off unless they are always on for all actions they effect. For Hindrances I went always on because a players more likely to remember to turn a bad one off than on. But I ended up not being consistent anyway (Can't swim didn't have always on because it comes up so rarely given how often Athletics is rolled). I'm OK with it going either way and Both the arguments of "It is used most of the time" and "Players are more likely to forget to turn it off than on"  apply. I'm not sure an absolute rule can be made for every edge. Sorry. Maybe leave it always on and GM's can adjust it if they want.
